### PR TITLE
make compatible and bump to LTS 12.24

### PR DESCRIPTION
**Pull request by ababkin**

### CODE REVIEW

File: kubernetes/kubernetes.cabal [hunk 1]
```diff
@@ -44,7 +44,7 @@ library
     , http-client >=0.5 && <0.6
     , http-client-tls
     , http-media >= 0.4 && < 0.8
-    , http-types >=0.8 && <0.12
+    , http-types >=0.8 && <0.13
```
Comment (guoshimin) [inline kubernetes/kubernetes.cabal]: This file is auto-generated by swagger-codegen. To make a change, we need to modify the source at https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/haskell-http-client/haskell-http-client.cabal.mustache.

**Comment by ababkin** [comment]
makes sense, thank you

**Comment by akshaymankar** [comment]
I PR-ed latest LTS to swagger-codegen: https://github.com/swagger-api/swagger-codegen/pull/9209 
Next release of codegen should get all the deps up to date 🎉

**Comment by ababkin** [comment]
great thanks

**Pull request closed without merging**